xgi/drawing/draw.py

```diff
@@ -985,3 +991,155 @@ def _update_lims(pos, ax):
+        The hyperedges transparency.
+    palette : None or array of colors (default=None)
+        Array of colors given by the hyperedges of different orders
+    
+    node_size : scalar or array (default=300)
+        Size of nodes.  If an array it must be the same length as nodelist.
+    
+    node_color : color or array of colors (default='tab:blue')
+        Node color. Can be a single color or a sequence of colors with the same
+        length as nodelist. Color can be string or rgb (or rgba) tuple of
+        floats from 0-1. If numeric values are specified they will be
+        mapped to colors using the cmap and vmin,vmax parameters. See
+        matplotlib.scatter for more details.
+    
+    node_shape : string (default='o')
+        The marker used for nodes, used in determining edge positioning.
+        Specification is as a `matplotlib.markers` marker, e.g. one of 'so^>v<dph8'.
+    
+    node_linewidths : None, scalar, sequence (default=1.0)
+        Line width of symbol border
+    
+    node_edgecolors : None, scalar, sequence (default = 'black')
+        Colors of node borders
+    
+    dyad_color : str, dict, iterable, or EdgeStat (default='black')
+        Color of the dyadic links.  If str, use the same color for all edges. If a dict, must
+        contain (edge_id: color_str) pairs.  If iterable, assume the colors are
+        specified in the same order as the edges are found in H.edges. If EdgeStat, use a colormap
+        (specified with dyad_color_cmap) associated to it
+
+    Returns
+    -------
+    ax : matplotlib.pyplot.axes
+
+    """
+    
+    if pos is None:
+        pos = barycenter_spring_layout(H)
+        
+    if ax is None:
+        ax = plt.gca()
+
+    ax.get_xaxis().set_ticks([])
+    ax.get_yaxis().set_ticks([])
+    ax.axis("off")
+    
+    if not max_order:
+        max_order = max_edge_order(H)
+    
+    if palette is None:
+        palette=sns.color_palette("deep", 6)
+    
+    for id, he in H.edges.members(dtype=dict).items():
+        d = len(he) - 1
+        if d > max_order:
+            continue
+        if d == 1:
+            # Drawing the edges
+            he = list(he)
+            x_coords = [pos[he[0]][0], pos[he[1]][0]]
+            y_coords = [pos[he[0]][1], pos[he[1]][1]]
+            line = plt.Line2D(
+                x_coords,
+                y_coords,
+                color=dyad_color,
+                zorder=1,
+                alpha = 1,
+            )
+            ax.add_line(line)
+            
+        else:
+            coordinates = [[pos[n][0], pos[n][1]] for n in he]
+            draw_hull(node_positions=np.array(coordinates), radius=radius, ax=ax, contour_color='tab:gray', filling_color=palette[d-1], alpha_filling=alpha_filling)
+    
+    G=convert_to_graph(H)
+    nx.draw_networkx_nodes(G, pos=pos, node_size=node_size, node_color=node_color, node_shape=node_shape, ax=ax, linewidths=node_linewidths, edgecolors=node_edgecolors, alpha=1)    
```

Comment:
Any particular reason not to use the `draw_nodes` from xgi instead of the networkx one? I'm just saying this for consistency with the other plotting functions

Reply:
Hi @iaciac, actually there was no reason for using the nx, that's why it is corrected in a later push (https://github.com/ComplexGroupInteractions/xgi/pull/320/commits/ca1451f1a5c348412868eec9a709f33d22b75bd1)

Reply:
Oooops sorry